### ISSUE TITLE
No request timeout causing points not being written to influxdb

### ISSUE THREAD

**Issue by jorisplusplus** [influxdata/influxdb-client-python#221]
We have deployed some IOT sensors and we are seeing some sensors not reporting values for periods of 15 min or more. I believe this is caused because there is no request_timeout used in the rest.py. In the debug logs of the program the last line is the post request being made and getting no response. This causes the influxdb client to stop writing points to the influxdb. 

I thought the timeout parameter of the InfluxDBClient is setting the request timeout? If not, is it possible to set the request timeout? Because it is supported in the rest.py file (lines 159-165)

__Steps to reproduce:__
Minimal piece of code I run to reproduce.
```python3
from influxdb_client import InfluxDBClient, Point
from influxdb_client.client.write_api import SYNCHRONOUS

client = InfluxDBClient(url="http://127.0.0.1:8080", token="token", org="org", debug=True, timeout=1000)
write_api = client.write_api(write_options=SYNCHRONOUS)

p = Point("point").tag("tag","tag").field("Field", 1.0)
write_api.write(bucket="bucket", record=p)
```
1. run nc -l -k -p 8080 (to host a tcp server for the client to connect to)
2. run code shown above

__Expected behavior:__
I would expect it to raise an exception, restart the connection or resend the post request

__Actual behavior:__
Post request is made and continues waiting until the program is restarted.

__Specifications:__
 - Client Version: 0.16.0
 - InfluxDB Version: N/A
 - Platform: Linux

**Comment by bednar** [comment]
@jorisplusplus, thanks for issue, We will take a look.

**Comment by bednar** [comment]
Hi @jorisplusplus,

The issue is fixed in [1.17.0](https://github.com/influxdata/influxdb-client-python/milestone/19) milestone.

If you would like to use a dev version then install client via:

pip install git+https://github.com/influxdata/influxdb-client-python.git@master

Regards

**Comment by jorisplusplus** [comment]
Thank you very much for the quick fix.

I will test it on some of the sensors, hopefully the problem is now solved.
